pyproject.toml

```diff
@@ -53,10 +51,6 @@ addopts = [
 markers = [
     "extended: mark test to be part of the extended test suite"
 ]
-filterwarnings =[
-    "ignore::FutureWarning:rpy2",
-    "ignore::DeprecationWarning:rpy2"
```

Comment:
why did you remove this? Just to double check is intended

Reply:
Thanks! Will bring it back